app/src/main/java/com/nervousfish/nervousfish/activities/SelectVerificationMethodActivity.java

```diff
@@ -53,11 +55,21 @@ public void openVerificationMethod(final View view) {
             case R.id.select_visual_verification:
                 LOGGER.info("Selected visual verification method, opening activity");
                 intent.setComponent(new ComponentName(this, VisualVerificationActivity.class));
```

Comment:
Can you move the `.setComponent` to after the try-catch block, similar to the `R.id.select_rhythm_verification` case 😇